config/pipeline.yaml

```
@@ -179,6 +179,15 @@ jobs:
   kbuild-gcc-10-arm64:
     <<: *kbuild-gcc-10-arm64-job
 
+  kbuild-gcc-10-arm64-dtbscheck:
+    <<: *kbuild-gcc-10-arm64-job
+    params:
```

```suggestion
    kind: test
    params:
```
`kbuild-gcc-10-arm64-job` has `kind: kbuild`, I think this one should be different

---

yes, that's what I meant, and also why we keep using the kbuild template; but the node/results should clearly be identified as `test` for easier understanding.